feconf.py

```diff
@@ -348,6 +349,14 @@ def get_empty_ratings():
     },
 }
 
+# Ids of the allowed classification algorithm classes.
+ALLOWED_CLASSIFIER_CLASSES = [{
```

Comment:
The reason we had 'ALLOWED' for interactions was because some interactions were being developed as test interactions and we didn't want them to be user-visible while they were being developed. Not sure if this is going to apply to classifiers, so I defer to y'all here.